workflows/diagnostics/fv3net/diagnostics/prognostic_run/shell.py

```diff
@@ -66,12 +66,15 @@ def __init__(self):
 
     def load(self, url):
         prognostic = load_run_data.SegmentedRun(url, catalog)
-        self.data_3d = prognostic.data_3d
-        self.data_2d = prognostic.data_2d.merge(grid.drop("area"))
+        self.data_3d = prognostic.data_3d.merge(grid)
+        self.data_2d = prognostic.data_2d.merge(grid, compat="override")
```

Comment:
The average plot still not working for my run. `compat='override'` will use the area from the left dataset, i.e. prognostic run data.